src/pudl/transform/epacems.py

```diff
@@ -189,13 +166,17 @@ def transform(
     # Create all the table inputs used for the subtransform functions below
 
     return (
-        raw_df.pipe(apply_pudl_dtypes, group="epacems")
-        .pipe(remove_leading_zeros_from_numeric_strings, "emissions_unit_id_epa")
```

Comment:
Did you verify that `strip_chars_start("0")` does exactly the same thing as `remove_leading_zeros_from_numeric_strings()`? If not we'll get some weird ID mismatches with the other data sources this is applied to.